### ISSUE TITLE
[BUG] Requirements conflicts 

### ISSUE THREAD

**Issue by tommylge** [Vinyzu/Botright#80]
```
2024-03-20 14:14:33.383 [info] INFO: pip is looking at multiple versions of undetected-playwright-patch to determine which version is compatible with other requirements. This could take a while.
2024-03-20 14:14:33.419 [info] ERROR: Cannot install -r /Users/xxx/Desktop/dev/repos/Botright/requirements.txt (line 4) and -r /Users/xxx/Desktop/dev/repos/Botright/requirements.txt (line 5) because these package versions have conflicting dependencies.
2024-03-20 14:14:33.419 [info] 
The conflict is caused by:
    playwright 1.42.0 depends on greenlet==3.0.3
    undetected-playwright-patch 1.40.0.post1700587210000 depends on greenlet==3.0.1
```

There's a conflict between playwright and undetected-playwright-patch regarding greenlet.
This is caused by https://github.com/kaliiiiiiiiii/undetected-playwright-python/blob/8886a371b5758dc59026c5fc0e8735ef8b754427/setup.py#L228
```
    install_requires=[
        "greenlet==3.0.1",
        "pyee==11.0.1",
        "typing-extensions;python_version<='3.8'",
    ],
```

I don't really know where to address this issue as it comes from `kaliiiiiiiiii` repo, but your requirements are then, triggering the conflict.

**Comment by Yorickpy** [comment]
yeah i have the same issue ims using python 3.11 windows already try with env and no env

**Comment by Vinyzu** [comment]
Im currently working on a undetected-playwright update...

**Comment by Yorickpy** [comment]
> Im currently working on a undetected-playwright update...



> Im currently working on a undetected-playwright update...

no problem take your time :) i think its a lot of work and you send this for free thats crazy tbh i will not hesitate if i can find a undetected browser lib that i can buy

**Comment by coffecup25** [comment]
Just wondering if you have an estimation for when the update is ready? I'm really impressed with what you're doing. Let me know if I can help in any way

**Comment by Vinyzu** [comment]
> Just wondering if you have an estimation for when the update is ready? I'm really impressed with what you're doing. Let me know if I can help in any way

Atm i hope ill get it ready next month (august)
